persistence.sh

```diff
@@ -157,7 +157,8 @@ persist_bridge_keys() {
 	upsert_file ${AWS_APPROVER_PASSWORD} ${VOLUMES_DIR}/volume3/approver/password ${AWS_PRIMARY_PROFILE}
 
 	approver_private_key=$(get_private_key ${VOLUMES_DIR}/volume3/approver)
-	upsert_secret ${AWS_APPROVER_PRIVATE_KEY} $approver_private_key ${AWS_SECONDARY_PROFILE}
+	upsert_secret ${CORE_ENV}/${AWS_APPROVER_PRIVATE_KEY} $approver_private_key ${AWS_PRIMARY_PROFILE}
```

Comment:
We don't want to bleed the `${CORE_ENV}` into this file, that should be coming from the `.env` file